config/routes.rb

```diff
@@ -0,0 +1,3 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+end
```

Comment:
Don't see any routes in here. You want be able to use any of the path helpers (e.g., `posts_path`) referenced in your application without them.

Quick fix to start you off...
```rb
resources :posts
```